Sources/HTMLEntities/String+HTMLEntities.swift

```diff
@@ -17,56 +17,74 @@
 /// This String extension provides utility functions to convert strings to their
 /// HTML escaped equivalents and vice versa.
 public extension String {
+    /// Global HTML escape options
+    public struct HTMLEscapeOptions {
```

Comment:
Did you consider making this conform to [OptionSet](https://developer.apple.com/reference/swift/optionset)? I think if you did then you could pass a `HTMLEscapeOptions` into `htmlEscape()` which would be nicer?